packages/cozy-device-helper/src/flagship.ts

```diff
@@ -15,7 +17,16 @@ export interface FlagshipMetadata {
   version?: string
 }
 
+const getGlobalWindow = (): Window => {
+  if (typeof window !== 'undefined') return window
+  else {
+    log('error', '"window" is not defined')
```

Comment:
```suggestion
    log('error', '"window" is not defined. This means that getGlobalWindow() shouldn't have been called and investigation should be done to prevent this call')
```